util/admin/admin.go

```diff
@@ -391,12 +391,13 @@ func compareDatumSlice(sc *stmtctx.StatementContext, val1s, val2s []types.Datum)
 		return false
 	}
 	for i, v := range val1s {
-		if v.Kind() == types.KindMysqlDecimal {
+		switch v.Kind() {
+		case types.KindMysqlDecimal, types.KindBytes:
```

Comment:
done. 
I used to think that using `CompareDatum` instead of `reflect.DeepEqual` may cause poor performance. I felt I was wrong after the benchmark test.